model_analyzer/result/run_config_measurement.py

```diff
@@ -293,7 +293,7 @@ def get_gpu_metric_value(self, tag, default_value=0):
         return default_value if self.get_gpu_metric(
             tag) is None else self.get_gpu_metric(tag).value()
 
-    def get_weighted_non_gpu_metric_value(self, tag, default_value=0):
+    def get_weighted_metric_value(self, tag, default_value=0):
```

Comment:
is there no longer a difference between non-gpu metrics and gpu-metrics?

Reply:
Good catch. This shouldn't have changed names (I originally thought I needed to, but turned out not to be the case). I've reverted this name.